data/serializer/src/main/java/tech/pegasys/teku/api/response/v1/validator/SyncCommitteeDuty.java

```diff
@@ -55,7 +56,7 @@
   public SyncCommitteeDuty(
       @JsonProperty("pubkey") final BLSPubKey pubkey,
       @JsonProperty("validator_index") final UInt64 validatorIndex,
-      @JsonProperty("validator_sync_committee_indices") final Set<Integer> committeeIndices) {
+      @JsonProperty("validator_sync_committee_indices") final IntSet committeeIndices) {
```

Comment:
Basically anything with `@JsonProperty` will need to stay declared using the `Integer` form to ensure our OpenAPI docs come out right.  We *might* be able to get away with actually passing it an `IntSet` as the implementation but I'm not sure about that - we'll have to test it.

Reply:
Good to know. I'll revert those instances.